fauna/manifest.json

```diff
@@ -0,0 +1,60 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "2be7cc0c-a21f-43ad-b2b7-3f41a98ad299",
+  "app_id": "fauna",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "uninstallation": "README.md#Uninstallation",
+    "changelog": "CHANGELOG.md",
+    "description": "Import your Fauna query logs into Datadog.",
+    "title": "Fauna",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Fauna Dashboard Metrics",
+        "image_url": "images/dashboard_metrics.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Fauna Dashboard Logs",
+        "image_url": "images/dashboard_logs.png"
```

Comment:
Could you include one more image of the integration? We typically request 3 images at least. 

 Additionally, is there a way you can populate data for the Dashboard logs image? We typically avoid having images of dashboards that have no data in the widgets.

Reply:
yep will get those added!